### ISSUE TITLE
Broken link to Widevine API details in Widevine tutorial

### ISSUE THREAD

**Issue by icbaker** [google/shaka-packager#785]
The top paragraph of [this page](https://google.github.io/shaka-packager/html/tutorials/widevine.html) says:

> Shaka Packager can talk to Widevine Cloud Service or any key server that implements [Common Encryption API for Widevine DRM](http://bit.ly/2vTG4oo) to fetch encryption keys.

That bit.ly address redirects to https://storage.googleapis.com/wvdocs/Widevine_DRM_Encryption_API.pdf

Which serves me the following XML response:

```xml
<Error>
<Code>AccessDenied</Code>
<Message>Access denied.</Message>
<Details>Anonymous caller does not have storage.objects.get access to the Google Cloud Storage object.</Details>
</Error>
```

**Comment by kqyang** [comment]
Thanks for letting us know. 

Here is the correct link: https://storage.cloud.google.com/wvdocs/Widevine_DRM_Encryption_API.pdf. Note that you may need to have a license agreement with Widevine to access it.

We'll update the doc.

**Comment by vish91** [comment]
@kqyang would you know if this doc url changed again ? 
```
https://storage.cloud.google.com/wvdocs/Widevine_DRM_Encryption_API.pdf
Request Method: GET
Status Code: 403 
```

**Comment by kqyang** [comment]
No. The link is correct. As mentioned above, you'll need to have a license agreement with Google Widevine to access it.

**Comment by vish91** [comment]
gotcha. thanks
